### PR TITLE
feat(ci): add a new tag in ci-visibility test for known flakes

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
Add a new flag to ci test uploaded through junit xml files

### Motivation
Improved management of flakes.

### CODE REVIEW

File: tasks/libs/junit_upload_core.py [hunk 1]
```diff
@@ -51,6 +54,18 @@ def junit_upload_from_tgz(junit_tgz, codeowners_path=".github/CODEOWNERS"):
     with open(codeowners_path) as f:
         codeowners = CodeOwners(f.read())
 
+    for test_file in TEST_OUTPUT_FILES:
+        if os.path.isfile(test_file):
+            with open(test_file) as f:
+                test_output = json.load(f)
+    else:
+        test_output = []
```
Comment (amenasria) [inline tasks/libs/junit_upload_core.py:62]: **🥜 nitpick:** ‏An indentation is needed here

**Review by amenasria — APPROVED** [review]
Few nits, otherwise LGTM !

File: tasks/libs/junit_upload_core.py [hunk 2]
```diff
@@ -24,6 +25,8 @@
 JOB_URL_FILE_NAME = "job_url.txt"
 JOB_ENV_FILE_NAME = "job_env.txt"
 TAGS_FILE_NAME = "tags.txt"
+TEST_OUTPUT_FILES = ["test_output.json", "test_output_fast.json"]
+FLAKE_MESSAGES = ["flakytest: this is a known flaky test", "flaky test in CI"]
```
Comment (pducolin) [inline tasks/libs/junit_upload_core.py:29]: **💬 suggestion**
We should track only the known and standard marker to push people to adopt it - we might want to always skip marked tests at some point, when we have a job to run only marked tests
```suggestion
FLAKE_MESSAGES = ["flakytest: this is a known flaky test"]
```

File: tasks/libs/junit_upload_core.py [hunk 3]
```diff
@@ -42,7 +45,72 @@ def add_flavor_to_junitxml(xml_path: str, flavor: AgentFlavor):
...
+    for test_file in TEST_OUTPUT_FILES:
+        if os.path.isfile(test_file):
+            with open(test_file) as f:
+                test_output = json.load(f)
+    else:
+        test_output = []
+    flaky_tests = [
+        "/".join([test["Package"], test["Test"]])
+        for test in test_output
+        if any(flake in test["Output"] for flake in FLAKE_MESSAGES)
```
Comment (pducolin) [inline tasks/libs/junit_upload_core.py:66]: **🥜 nitpick**
```suggestion
        if any(flakeMessage in test["Output"] for flakeMessage in FLAKE_MESSAGES)
```

File: tasks/libs/junit_upload_core.py (hunk 2 above)
Comment (chouetz) [reply tasks/libs/junit_upload_core.py:29]: I'm ok with that, then we should probably [replace this](https://github.com/DataDog/datadog-agent/blob/main/pkg/process/procutil/process_linux_test.go#L1397) by a call to `flake.Mark`, wdyt?

File: tasks/libs/junit_upload_core.py (hunk 1 above)
Comment (chouetz) [reply tasks/libs/junit_upload_core.py:62]: wow this is not a nit, this is a correction I need to perform, many thanks!!

File: tasks/libs/junit_upload_core.py (hunk 2 above)
Comment (chouetz) [reply tasks/libs/junit_upload_core.py:29]: Going back to just tracking the `flake.Mark` message

**Pull request merged**